README.md

```diff
@@ -33,7 +33,17 @@ Anna is part of KWOC : You can talk to us on Zulip or comment on issues to start
   - If I don't understand
   - Close tabs
   
-
+ # How to install ANNA
+ 
+1. Once you are on the repository page,look for the green “Clone or download” button on the right-hand side of the page.
+![imageedit_3_3326230850](https://user-images.githubusercontent.com/29461633/33730973-a91d3c7c-dba7-11e7-8efe-9d1b9f43980b.png)
+2. Click the “Clone or download” button and then click “Download ZIP.”
+![screenshot from 2017-12-07 20_02_51-iloveimg-cropped](https://user-images.githubusercontent.com/29461633/33731004-c02d137e-dba7-11e7-988b-c1527d724896.png)
+3. Take the downloaded ZIP file and extract it. We need to give it a permanent home because Chrome will externally load the files (if they get moved/deleted then the extension will stop working).
```

Comment:
from step 3 can you make like this .

3. Using the cloned folder. Go to chrome menu > more tools extensions or chrome:/extensions 
4. Click on load unpacked extension and select the folder and install.
![deepinscreenshot_select-area_20171208003514](https://user-images.githubusercontent.com/12110844/33733571-2a403612-dbb0-11e7-8d21-d511f811c157.png)
5. Make sure to give Anna microphone permissions and you are good to go :tada: .